components/Banner/Banner.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import { useEffect, useState } from 'react';
+import { X } from 'lucide-react';
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+
+const Banner = () => {
+    const [visible, setVisible] = useState(true);
+    const pathname = usePathname();
+    const [isHomePage, setIsHomePage] = useState(false);
+
+    useEffect(() => {
+        setIsHomePage(pathname === '/');
+    }, [pathname]);
+
+    if (!visible || !isHomePage) return null;
```

Comment:
Do we want to show the banner in only the homepage ?

Reply:
Yes, this is intentional. The banner is supposed to be only visible on the homepage.